sdk/eventhub/event-hubs/src/checkpointManager.ts

```diff
@@ -49,8 +50,11 @@ export class CheckpointManager {
   private _eTag: string;
 
   /**
-   * @ignore
-   * @interal
```

Comment:
Why was this removed?
We don't want to document the constructor as we don't want to encourage users to try and create the check point mangers on their own

I would add the `@private` and `@ignore` tags